app/src/organisms/Desktop/Devices/ProtocolRun/SetupCamera/index.tsx

```diff
@@ -76,6 +80,20 @@ export function SetupCamera({
     })
     confirmCameraSettings()
   }
+  const baseProps = {
+    source: 'runRecord' as const,
+    robotType: robotType,
+  }
+  const { reportPhotoAccessUsage } = useCameraAnalytics(baseProps)
+  useEffect(() => {
+    if (storageInfo.isImageStorageLow) {
+      reportPhotoAccessUsage({
+        ...baseProps,
+        runId: runId,
+        action: 'storageWarning',
+      })
+    }
+  })
```

Comment:
```suggestion
  }, [storageInfo.isImageStorageLow !== null])
```
We need to update the dependency array so the useEffect fires when we have the correct data.